paasta_tools/cli/cmds/start_stop_restart.py

```diff
@@ -139,19 +131,48 @@ def paasta_start_or_stop(args, desired_state):
     soa_dir = args.soa_dir
     service = figure_out_service_name(args=args, soa_dir=soa_dir)
 
-    branch = "paasta-%s.%s" % (cluster, instance)
-    all_branches = list(get_branches(service=service, soa_dir=soa_dir))
+    instance_type = utils.validate_service_instance(
+        service=service,
+        instance=instance,
+        cluster=cluster,
+        soa_dir=soa_dir,
+    )
+
+    if instance_type == 'marathon':
+        service_config = load_marathon_service_config(
+            service=service,
+            instance=instance,
+            cluster=cluster,
+            soa_dir=soa_dir,
+            load_deployments=False,
+        )
+    elif instance_type == 'chronos':
+        service_config = load_chronos_job_config(
+            service=service,
+            instance=instance,
+            cluster=cluster,
+            soa_dir=soa_dir,
+            load_deployments=False,
+        )
+    else:
+        print "A service config other than type 'marathon' or 'chronos' was loaded!"
+        print "'%s' is not currently supported by paasta %s" % (instance_type, desired_state)
+        sys.exit(3)
```

Comment:
Yuck. We already have almost literally this same code in 
paasta_tools/cli/cmds/local_run.py:550
Can you move that to utils and share that instead?